remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -97,6 +100,13 @@ public void testGetTags() {
         assertEquals(expectedTags, actualCreatedTags);
     }
 
+    @Test
+    public void testTagNotFoundValidation()
+    {
+        given(nodeServiceMock.hasAspect(any(),any())).willReturn(true);
+        assertThrows(EntityNotFoundException.class, () ->objectUnderTest.validateTag(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, TAG_ID));
```

Comment:
It's not clear to me why this method should throw the EntityNotFoundException. Please add comments or some more suggestive mock methods to explain this.

Reply:
Remove the test case as there was a preexisting EntityNotFoundException test case. So removed the duplicate test case